packages/components/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ### Enhancements
 
 -   `ToolbarButton`: Deprecate `isDisabled` prop and merge with `disabled` ([#63101](https://github.com/WordPress/gutenberg/pull/63101)).
+-   `Button`: Stabilize `__experimentalIsFocusable` prop ([#62282](https://github.com/WordPress/gutenberg/pull/62282)).
```

Comment:
Is the stable prop name worth a mention in the CHANGELOG? Could be useful if someone is researching when that prop was introduced.

Perhaps like this:

```suggestion
-   `Button`: Stabilize `__experimentalIsFocusable` prop as `accessibleWhenDisabled` ([#62282](https://github.com/WordPress/gutenberg/pull/62282)).
```